oaknational/Oak-Web-Application
# Added "lot picker" to curric homepage

## Description
Added "lot picker" to curriculum homepage

**Note**: Changing the photo and loop-line will be picked up in another ticket.

## Issue(s)

Fixes `CUR-1140`

## How to test

1. Go to https://deploy-preview-3076--oak-web-application.netlify.thenational.academy/curriculum
2. Check picker is now displayed and works correctly
3. Check visuals are as outlined in `CUR-1140`
